app/views/directives/create-secret.html

```diff
@@ -70,6 +72,16 @@
           Value of the secret will be supplied when invoking the webhook.
           <a ng-href="{{'webhooks' | helpLink}}" target="_blank"><span class="learn-more-inline">Learn More&nbsp;<i class="fa fa-external-link" aria-hidden="true"></i></span></a>
         </div>
+        <div class="has-error" ng-show="secretForm.webhookSecretKey.$error.pattern && secretForm.webhookSecretKey.$touched">
+          <span class="help-block">
+            {{secretReferenceValidation.description}}
+          </span>
+        </div>
+        <div class="has-error" ng-show="secretForm.webhookSecretKey.$error.minlength && secretForm.webhookSecretKey.$touched">
+          <span class="help-block">
+            Minimum character count is {{secretReferenceValidation.minLength}}.
```

Comment:
looks like you got this message from the env editor, but I prefer what we say in other places which is something like:

`The key must have at least 8 characters.`